client/init.c

```diff
@@ -249,6 +233,25 @@ int parse_startup_parameters(int argc, char **argv, struct startup_parameters *p
 		}
 	}
 
+	if(parameters->pool_address != NULL && (parameters->pool_configuration.node_address != NULL || parameters->pool_configuration.mining_configuration != NULL)) {
+		printf("{0} must be started either as pool or as wallet.\n", argv[0]);
```

Comment:
why {0} ?

Reply:
Ah, my bad. {0} - this syntax is from C# :)

Reply:
Fixed